src/iris/api.py

```diff
@@ -1718,13 +1718,13 @@ def on_post(self, req, resp):
                     session.commit()
                     session.close()
                 except (InternalError, OperationalError) as e:
-                    logger.exception('Failed inserting incident. (Try %s/%s)', retries, max_retries)
+                    logger.error('Failed inserting incident. (Try %s/%s)', retries, max_retries)
```

Comment:
Since we're only logging the error message now (no stack trace), we should also log the `plan_id` or something additional to give context as to what incident we're talking about when we see this log message in the logs.

Reply:
sure